agent/socket.go

```diff
@@ -190,3 +193,22 @@ func (s *Server) extendPod(ctx context.Context, channel, namespace, pod string,
 	}
 	return nil
 }
+
+func (s *Server) updateDeletionTime(ctx context.Context, namespace, pod string, tm time.Time) error {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return err
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
```

Comment:
I think you can reuse clientset too.

Reply:
Thank you.
I am sorry for the same comment.
I have not been aware of the reuse of resources.